interfaces/acados_template/acados_template/generate_solver.py

```diff
@@ -310,220 +371,6 @@ def render_template(in_file, out_file, template_dir):
     os.system('make shared_lib')
     os.chdir('..')
 
-    ## Load solver
-    solver = acados_solver(
-        acados_ocp,
-        'c_generated_code/libacados_solver_{model_name}.so'.format(model_name=model.name)
-    )
-    return solver
-
-class acados_solver:
```

Comment:
I like putting that into another!
Make sure, the setters down there are in the other file in latest version.